.github/workflows/tests_linux.yml

```diff
@@ -486,4 +486,61 @@ jobs:
         uses: codecov/codecov-action@v3
         with:
           fail_ci_if_error: true
-          token: ${{ secrets.CODECOV_TOKEN }}
\ No newline at end of file
+          token: ${{ secrets.CODECOV_TOKEN }}
+
+  cpptestsWithMultipleBackends:
+    strategy:
+      matrix:
+        os: [ubuntu-22.04]
+        exec_model: [\"SERIAL\"]
```

Comment:
We build with both Serial and OpenMP execution kernels, however, I think Serial is sufficient for testing the compilation of multiple backends in this repo. Let me know if you think otherwise.

Reply:
I think it is better to have it in the most similar to Catalyst way possible. 
In this case, I would prefer to have both been tested here.